src/components/Slider/RangeSlider/RangeSlider.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import {
+  Root as SliderRoot,
+  Track as SliderTrack,
+  Range as SliderRange,
+  Thumb as SliderThumb
+} from '@radix-ui/react-slider';
+
+import styles from '../Slider.module.scss';
+import classNames from 'classnames/bind';
+const cx = classNames.bind(styles);
+
+export interface RangeSliderProps {
+  values?: number[];
+  min: number;
+  max: number;
+  step: number;
+  minStep?: number;
+  isSmall?: boolean;
+  isLight?: boolean;
```

Comment:
Don't need to worry about theming - apologies, this should have been mentioned in the spec.

Reply:
@colbr As in should I just remove the isLight property and associated code or?

Reply:
Yep you can remove `isLight` and all of the associated code - our theme engine creates CSS variables so no need to handle it in JS, you just use the CSS variables and they'll change based on the theme.

Reply:
oh perfect